automation/e2e-pipelines/example-pipelines-test.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+function wait_until_exists() {
+  timeout 10m bash <<- EOF
+  until oc get $1; do
+    sleep 5
+  done
+EOF
+}
+
+function wait_for_pipelinerun() {
+  oc wait -n ${namespace} --for=condition=Succeeded=True pipelinerun -l pipelinerun="$1"-run --timeout=60m &
+  success_pid=$!
+
+  oc wait -n ${namespace} --for=condition=Succeeded=False pipelinerun -l pipelinerun="$1"-run --timeout=60m && exit 1 &
+  failure_pid=$!
+
+  wait -n $success_pid $failure_pid
+
+  if (( $? == 0 )); then
+    echo "Pipelinerun $1 succeeded"
+  else
+    echo "Pipelinerun $1 failed"
+    exit 1
+  fi
+}
+
+cp -L "$KUBECONFIG" /tmp/kubeconfig && export KUBECONFIG=/tmp/kubeconfig
+export DEPLOY_NAMESPACE=kubevirt
+
+# switch to faster storage class for example pipelines tests (slower storage class is causing timeouts due 
+# to not able to copy whole windows disk)
+if ! oc get storageclass | grep -q 'ssd-csi (default)' > /dev/null; then
+  oc annotate storageclass ssd-csi storageclass.kubernetes.io/is-default-class=true --overwrite
+  oc annotate storageclass standard-csi storageclass.kubernetes.io/is-default-class- --overwrite
+fi
+
+# Deploy resources
+echo "Deploying resources"
+./automation/e2e-deploy-resources.sh
+
+# remove tsc node labels which causes that windows VMs could not be scheduled due to different value in tsc node selector
```

Comment:
Is that still necessary in newer versions of KubeVirt?

Reply:
It looks like it is no longer needed. Removed